<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="BotonAtras">
      <property name="text">
       <string>  Regresar</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::EditUndo"/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MÃ©todo Iterativo</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="BotonInfo">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ingrese los datos del problema y luego haga clic en calcular</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Datos:</string>
      </property>
      <widget class="QDoubleSpinBox" name="inputVSS">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>68</width>
         <height>26</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>40</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="inputR">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>68</width>
         <height>26</height>
        </rect>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
       <property name="singleStep">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_3">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="inputN">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>68</width>
         <height>26</height>
        </rect>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_4">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputIs">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhFormattedNumbersOnly|Qt::InputMethodHint::ImhPreferNumbers</set>
       </property>
       <property name="placeholderText">
        <string comment="1e-12"/>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_5">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>40</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="inputID">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>80</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhFormattedNumbersOnly|Qt::InputMethodHint::ImhPreferNumbers</set>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_6">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>80</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="inputVD">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>120</y>
         <width>68</width>
         <height>26</height>
        </rect>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.700000000000000</double>
       </property>
      </widget>
      <widget class="QToolButton" name="BotonInfo_7">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>120</y>
         <width>24</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::HelpAbout"/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltaje (Vss)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>80</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Resistencia (R)*</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>120</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Coeficiente (n)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Corriente Sat. (Is)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Corriente Inicial (iD)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>120</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltaje Inicial (vD)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>(V)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>(Î©)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>40</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>(A)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>80</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>(mA)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>120</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>(V)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Los campos con (*) son obligatorios</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnCalcular">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QPushButton" name="btnLimpiar">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" rowspan="2" colspan="2">
     <widget class="QTableWidget" name="tablaResultados">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>311</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="accessibleIdentifier">
       <string/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::PenStyle::SolidLine</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">IteraciÃ³n</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">vD (V)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">iD (mA)</string>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="2" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SoluciÃ³n:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLabel" name="labelResultado">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuElectr_nica_B_sica">
    <property name="title">
     <string>ElectrÃ³nica BÃ¡sica</string>
    </property>
   </widget>
   <addaction name="menuElectr_nica_B_sica"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
